modules/quant_purger/src/Entity/QuantPurgeSettings.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Drupal\quant_purger\Entity;
+
+use Drupal\purge\Plugin\Purge\Purger\PurgerSettingsBase;
+use Drupal\purge\Plugin\Purge\Purger\PurgerSettingsInterface;
+
+/**
+ * Defines the QuantPurgerSettings entity.
+ *
+ * @ConfigEntityType(
+ *   id = "quantpurgersettings",
```

Comment:
Id is quantpurgersettings (with an "r") but the class is QuantPurgeSettings (without an "r") o_O